### PR TITLE
Remove manual repository setup quirks

### PR DESCRIPTION
**Pull request by dirkmueller**
This shouldn't be necessary anymore - use the ardana support
for forwarding repositories instead.

### CODE REVIEW

File: scripts/jenkins/ardana/ansible/repositories.yml [hunk 1]
```diff
@@ -62,10 +42,10 @@
...
       - name: SLES12-SP3-Updates
         src:  SLES12-SP3-Updates
-      - name: SUSE-OpenStack-Cloud-8-Pool
-        src: SUSE-OpenStack-Cloud-8-Pool
-      - name: SUSE-OpenStack-Cloud-8-Updates
-        src: SUSE-OpenStack-Cloud-8-Updates
+        # - name: SUSE-OpenStack-Cloud-8-Pool
+        # src: SUSE-OpenStack-Cloud-8-Pool
+        # - name: SUSE-OpenStack-Cloud-8-Updates
+        # src: SUSE-OpenStack-Cloud-8-Updates
```
Comment (cmurphy) [inline scripts/jenkins/ardana/ansible/repositories.yml]: Why remove these? They still need to be mounted in order for ardana to pick them up. Although I guess they're not really needed as long as the development media is there...

Also, why comment these out instead of removing them?

File: scripts/jenkins/ardana/ansible/repositories.yml [hunk 2]
```diff
@@ -75,8 +55,8 @@
       - Cloud
       - SLES12-SP3-Pool
       - SLES12-SP3-Updates
-      - SUSE-OpenStack-Cloud-8-Pool
-      - SUSE-OpenStack-Cloud-8-Updates
+        # - SUSE-OpenStack-Cloud-8-Pool
+        # - SUSE-OpenStack-Cloud-8-Updates
```
Comment (cmurphy) [inline scripts/jenkins/ardana/ansible/repositories.yml]: Why comment these out instead of removing them?

File: scripts/jenkins/ardana/ansible/init.yml [hunk 3]
```diff
@@ -157,16 +147,6 @@
...
-  # FIXME: This can go away when https://suse-jira.dyndns.org/browse/SCRD-2479
-  #        is addressed.
-  - name: Switch designate to use bind in Input Model
-    replace:
-      dest: /var/lib/ardana/openstack/my_cloud/definition/data/control_plane.yml
-      regexp: 'powerdns'
-      replace: 'bind'
-    tags:
-      - create-ardana-input-model
-
```
Comment (cmurphy) [inline scripts/jenkins/ardana/ansible/init.yml]: (nit) This is unrelated to the subject of the commit message

File: scripts/jenkins/ardana/ansible/init.yml [hunk 4]
```diff
@@ -191,9 +171,6 @@
     become: true
     become_user: ardana
 
-  #- name: Output of config-processor-run.yml playbook
-  #  debug: var
-
```
Comment (cmurphy) [inline scripts/jenkins/ardana/ansible/init.yml]: (nit) This is unrelated to the subject of the commit message

File: scripts/jenkins/ardana/ansible/repositories.yml (hunk 1 above)
Comment (dirkmueller) [reply scripts/jenkins/ardana/ansible/repositories.yml]: exactly, the logic is that we need to avoid them being available for the case that we use the devel media (to make sure that *only* the packages from the devel media are used, not some othe rpackage that accidentally has a higher version number in the pool/update channel).

I can remove that if you think commenting out is bad for some reason.

File: scripts/jenkins/ardana/ansible/repositories.yml (hunk 2 above)
Comment (dirkmueller) [reply scripts/jenkins/ardana/ansible/repositories.yml]: I guess we need to conditionalize it at some point so the repository names are still useful imho

File: scripts/jenkins/ardana/ansible/init.yml (hunk 3 above)
Comment (dirkmueller) [reply scripts/jenkins/ardana/ansible/init.yml]: splitted into a separate commit

File: scripts/jenkins/ardana/ansible/init.yml (hunk 4 above)
Comment (dirkmueller) [reply scripts/jenkins/ardana/ansible/init.yml]: splitted into a separate commit
